Dockerfile

```
@@ -0,0 +1,20 @@
+FROM opensciencegrid/software-base:release
+
+LABEL maintainer OSG Software <help@opensciencegrid.org>
```

```suggestion
ARG BASE_YUM_REPO=release

FROM opensciencegrid/software-base:3.5-el7-$BASE_YUM_REPO

LABEL maintainer OSG Software <help@opensciencegrid.org>

ARG BASE_YUM_REPO=release
```

We want to be able to specify which Yum repos/software base to use using Docker build args